### PR TITLE
Drafting docs for local setup

### PR DESCRIPTION
**Pull request by dimpar**
This PR adds:

- `local-t-networks.adoc` includes instructions on how to run local
  implementation of the Ethereum protocol - Geth
- `README.adoc` includes general information for setting up a local
  environment to run T clients. It includes `./install.sh` & `./initialize.sh` scripts.

### CODE REVIEW

File: docs/development/README.adoc [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  Solidity code generally adheres to the
+  https://solidity.readthedocs.io/en/latest/style-guide.html[Solidity style guide].
+  Contracts and their functions are documented using
+  https://docs.soliditylang.org/en/develop/natspec-format.html[the
+  Ethereum Natural Specification Format] (NatSpec).
+
+== Working with Solidity contracts
+
+The fastest and easiest way to have a local Ethereum testent is to use Hardhat.
+https://hardhat.org/
```
Comment (pdyraga) [inline docs/development/README.adoc:68]: Should we make the word "Hardhat" a link? Now it looks a bit weird when rendered:

image

File: docs/development/README.adoc [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+The `+initialize.sh+` script will:
+
+* mint & approve `T` tokens
+* stake `T` tokens
+* increase authorization for `RandomBeacon`
+* register operator
+
+Please refer to the scripts `help` for available command arguments and environment variables. `./install.sh --help` or `./initialize.sh --help`. Accounts unlocking and staking can be run outside of these scripts as standalone hardhat tasks.
+
+To start the installation:
```
Comment (pdyraga) [inline docs/development/README.adoc:37]:
```suggestion
To deploy and initialize contracts:
```

File: docs/development/README.adoc [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+* run `./scripts/install.sh`. You need to run it only once.
+
+* run `./scripts/initialize.sh --stake-owner <address>` for the default setup for each client. Default setup assumes that the staking provider, operator, beneficiary and an authorizer are all have the same addresses as the stake owner. Only the `stake-owner` has to be unique for the default setup. The default stake amount is `1,000,000 T` tokens. The default authorization amount is a min. authorization amount \n.
```
Comment (pdyraga) [inline docs/development/README.adoc:41]: Was it expected that `\n` breaks a line? Because if so, it does not:

Comment (pdyraga) [inline docs/development/README.adoc:41]:
```suggestion
* run `./scripts/initialize.sh --stake-owner <address>` for the default setup for each client. Default setup assumes that the staking provider, operator, beneficiary, and the authorizer all have the same addresses as the stake owner. Only the `stake-owner` has to be unique for the default setup. The default stake amount is `1,000,000 T` tokens. The default authorization amount is the minimum authorization amount \n. 
```

File: docs/development/README.adoc (hunk 1 above)
Comment (dimpar) [reply docs/development/README.adoc]: [4ed08aa](https://github.com/keep-network/keep-core/pull/3050/commits/4ed08aafc06d8cb3c320094abee5fe331e66bf3c)

File: docs/development/README.adoc (hunk 2 above)
Comment (dimpar) [reply docs/development/README.adoc]: [8c6255f](https://github.com/keep-network/keep-core/pull/3050/commits/8c6255f0b1af65fcd691c19486371ca610f9f280)

File: docs/development/README.adoc (hunk 3 above)
Comment (dimpar) [reply docs/development/README.adoc]: [8c6255f](https://github.com/keep-network/keep-core/pull/3050/commits/8c6255f0b1af65fcd691c19486371ca610f9f280)

File: docs/development/README.adoc [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+* increase authorization for `RandomBeacon`
+* register operator
+
+Please refer to the scripts `help` for available command arguments and environment variables. `./install.sh --help` or `./initialize.sh --help`. Accounts unlocking and staking can be run outside of these scripts as standalone hardhat tasks.
+
+To deploy and initialize contracts:
+
+* run `./scripts/install.sh`. You need to run it only once.
+
+* run `./scripts/initialize.sh --stake-owner <address>` for the default setup for each client. Default setup assumes that the staking provider, operator, beneficiary, and the authorizer are all have the same addresses as the stake owner. Only the `stake-owner` has to be unique for the default setup. The default stake amount is `1,000,000 T` tokens. The default authorization amount is the minimum authorization amount +
```
Comment (pdyraga) [inline docs/development/README.adoc:41]: *are all have* or just *all have*?

Comment (pdyraga) [reply docs/development/README.adoc:41]: Also, can we break the line to keep the 80 margin?

Comment (dimpar) [reply docs/development/README.adoc]: [6865fcf](https://github.com/keep-network/keep-core/pull/3050/commits/6865fcf9931fd525748866cee3ef41649a67b7b7)

File: docs/development/README.adoc [hunk 5]
```diff
@@ -32,14 +32,22 @@ The `+initialize.sh+` script will:
...
 * run `./scripts/install.sh`. You need to run it only once.
 
-* run `./scripts/initialize.sh --stake-owner <address>` for the default setup for each client. Default setup assumes that the staking provider, operator, beneficiary, and the authorizer are all have the same addresses as the stake owner. Only the `stake-owner` has to be unique for the default setup. The default stake amount is `1,000,000 T` tokens. The default authorization amount is the minimum authorization amount +
-Ex. if you need to initialize a network with 3 clients with the default setup, then run this script 3 times with a different `<address>`.
+* run `./scripts/initialize.sh --stake-owner <address>` for the default setup for
+each client. Default setup assumes that the staking provider, operator, beneficiary,
+and the authorizer all have the same addresses as the stake owner. Only the
+`stake-owner` has to be unique for the default setup. The default stake amount is
+`1,000,000 T` tokens. The default authorization amount is the minimum authorization
+amount +
```
[the remark this answers is not in the document]
Comment (pdyraga) [reply docs/development/README.adoc:48]: For the next time, we'll be working on docs.

**Review by pdyraga — APPROVED** [review]

**Pull request merged**